security-questionnaire.md

```diff
@@ -2,88 +2,88 @@
 
-
+No. The API operates solely on file hashes and does not handle or expose any personal or identifiable user information.
 
 ## 04. How do the features in your specification deal with sensitive information?
 
-
+Files can only be accessed with explicit user consent. The API does not allow arbitrary file discovery or sharing of sensitive user data without permission.
 
 ## 05. Does data exposed by your specification carry related but distinct information that may not be obvious to users?
 
-
+No. The API design ensures that only the explicit file hash and its availability are exposed, with no hidden or derived information.
 
 ## 06. Do the features in your specification introduce state that persists across browsing sessions?
 
-
+Yes. Files stored in COS persist across sessions. However, their access is gated by user consent, and user agents can manage eviction policies to maintain control over this state.
 
 ## 07. Do the features in your specification expose information about the underlying platform to origins?
 
-
+No. The API does not provide any platform-specific details to origins.
 
 ## 08. Does this specification allow an origin to send data to the underlying platform?
 
-
+No. The API strictly enables storage and retrieval of files identified by hashes without direct interaction with the underlying platform.
 
 ## 09. Do features in this specification enable access to device sensors?
 
-
+No.
 
 ## 10. Do features in this specification enable new script execution/loading mechanisms?
 
-
+No.
 
 ## 11. Do features in this specification allow an origin to access other devices?
 
-
+No.
 
 ## 12. Do features in this specification allow an origin some measure of control over a user agent's native UI?
 
-
+No. The user agent remains in full control, particularly for displaying permission prompts.
 
 ## 13. What temporary identifiers do the features in this specification create or expose to the web?
 
-
+None. File access is based solely on static hashes, which are not session-specific identifiers.
 
 ## 14. How does this specification distinguish between behavior in first-party and third-party contexts?
 
-
+The permission model ensures that file access is user-controlled, regardless of the context. Explicit user consent is required for cross-origin access.
 
 ## 15. How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?
 
-
+Files stored in COS are not accessible in Private Browsing or Incognito mode unless explicitly allowed by the user agent. Data stored during such sessions is not retained.
 
 ## 16. Does this specification have both "Security Considerations" and "Privacy Considerations" sections?
 
-
+Yes. The specification includes detailed sections addressing security and privacy implications, including hashing, user consent, and eviction policies.
```

Comment:
We don't have these sections yet.

Reply:
Da beißt sich die Katze in den Schwanz. https://github.com/explainers-by-googlers/template/pull/5 introduces this in the explainer template. I'll work on this after we're happy with the questionnaire, as this section, essentially, just is a "best of" from the questionnaire answers.